src/skmultiflow/data/random_rbf_generator.py

```diff
@@ -150,10 +154,10 @@ def _prepare_for_use(self):
 
     def _generate_centroids(self):
         """ Generates centroids
-        
-        Sequentially creates all the centroids, choosing at random a center, 
-        a label, a standard deviation and a weight. 
-        
+
+        Sequentially creates all the centroids, choosing at random a center
```

Comment:
There is a missing comma in the documentation.

Reply:
Done!